src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  background-color: #ffff;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: row;
```

Comment:
it's default flex-flow value, you can omit it

Reply:
![image](https://user-images.githubusercontent.com/38065883/148239975-ac4d1bba-8f71-466b-b6df-8b693777de39.png)
BTW, there should be wrap also, so you don't need 9 line